snapcraft.yaml

```diff
@@ -97,3 +97,16 @@ parts:
       - -usr/share/upstart
       - -usr/share/zsh
       - -var
+
+  # Create a symlink xdg-email -> xdg-open, as the latter is perfectly able to
+  # handle mailto: URLs (see https://launchpad.net/bugs/1849774).
+  # xdg-open is a wrapper provided by the core snap.
+  xdg-email:
+    plugin: nil
+    override-pull: "" 
+    override-prime: |
+      set -eux
+      mkdir -p usr/bin
+      cp -r /snap/core22/current/usr/bin/xdg-open usr/bin
+      cd usr/bin
+      ln -s xdg-open xdg-email
```

Comment:
Because chromium snap got a [special permission](https://git.launchpad.net/review-tools/commit/?id=31c894178a8fd110b9f65f0ee532b7e11384f8b8) to do that.

Now, `xdg-open` in snaps is different, it's just a wrapper to the command

```bash
soumyaxubuntu@soumyas-gf63-thin-11uc:~$ cat /usr/bin/xdg-open
#!/bin/sh
exec snapctl user-open "$@"
```

This is in the snaps. Now, this is only in the `core22`, so, no stage-packages can help it. I'll try to add the same PR to the chromium snap also. But, here it's the best and cleanest implementation.

Reply:
Done